app/_components/Article/index.tsx

```
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import type { News } from "@/app/_libs/microcms";
+import Date from "../Date";
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)